src/main/java/pl/whitelines/configuration/SecurityConfig.java

```diff
@@ -5,44 +5,57 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
 
 @Configuration
 @EnableWebSecurity
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
-
 	@Autowired
 	private DataSource dataSource;
 	
+	@Autowired
+    public void configAuthentication(AuthenticationManagerBuilder auth) throws Exception {
+        auth.jdbcAuthentication()
+        		.dataSource(dataSource)
+        		.passwordEncoder(passwordEncoder())
+                .usersByUsernameQuery("select login, password, true enabled from users where login=?")
+                .authoritiesByUsernameQuery("select login, role from users where login=?");
+    }
 	
-	 @Autowired
-	    public void configAuthentication(AuthenticationManagerBuilder auth) throws Exception {
-	        auth.jdbcAuthentication()
-	        		.dataSource(dataSource)
-	        		.passwordEncoder(passwordEncoder())
-	                .usersByUsernameQuery("select login, password, true enabled from users where login=?")
-	                .authoritiesByUsernameQuery("select login, 'ROLE_USER' as role from users where login=?");
-	    }
+    @Bean
+    @Override
+    public AuthenticationManager authenticationManager() throws Exception {
+        return super.authenticationManager();
+    }
+
+    @Bean
+    public AuthenticationFilter authenticationFilter() throws Exception {
+        AuthenticationFilter authenticationFilter = new AuthenticationFilter();
+        authenticationFilter.setAuthenticationManager(authenticationManager());
+        return authenticationFilter;
+    }
 	
 	@Override
     protected void configure(HttpSecurity http) throws Exception {
-		 http.csrf().disable()
-			.authorizeRequests()
-				.anyRequest().authenticated()
-				.and()
-			.formLogin().and()
-			.httpBasic();
+		 http
+				.authorizeRequests()
+				.antMatchers("/notes/**").authenticated()
+				.and().formLogin().disable()
+				.httpBasic()
+				.and().addFilterBefore(authenticationFilter(), UsernamePasswordAuthenticationFilter.class)
+				.csrf().disable();
     }
-	

```

Comment:
Why you delete it?

Reply:
Because in this line was white chars which should not be here